pages/common/xml-validate.md

```diff
@@ -0,0 +1,24 @@
+# xml-validate
+
+> Validate XML documents.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Validate XML document for well-formedness only (default):
+
+`xml validate --well-formed {{path/to/input1.xml|URI}} {{input2.xml ...}}`
+
+- Validate XML document against Document Type Definition (DTD):
+
+`xml validate --dtd {{path/to/schema.dtd}} {{path/to/input1.xml|URI}} {{input2.xml ...}}`
+
+- Validate XML document against XML Schema Definition (XSD):
```

Comment:
Perhaps this will read better?
```suggestion
- Validate an XML document against an XML Schema Definition (XSD):
```

Reply:
- Validate XML document(s) against an XML Schema Definition (XSD):